packages/storybook-ui-components/stories/Tooltip.stories.tsx

```diff
@@ -160,12 +160,12 @@ export const Demo = () => (
             "bottom-start",
             "bottom-end",
           ],
-          "bottom",
+          "bottom"
```

Comment:
Trailing commas should also be handled by eslint. Make sure your editor is configured to use it.

Reply:
Done